renovate.json

```diff
@@ -29,7 +29,8 @@
       ],
       "matchPackageNames": [
         "django"
-      ]
+      ],
+      "allowedVersions": ">=4"
```

Comment:
```suggestion
      "allowedVersions": "<4"
```

Reply:
Isn't `"enabled": false,` inverting the filter ?

Reply:
Should we remove the the `enable:false` then ?

Reply:
We should rename this block to be specific to django, remove the `enable:false` and allow version `<4`.